config/locales/en.yml

```diff
@@ -462,11 +462,11 @@ en:
         edit_closure: 'Edit closure details'
 
     hint:
-      offender_sar_case_form:
-        prison_number: "You can include multiple numbers separated by comma, space or paragraph return."
-        subject_aliases: "You can include multiple aliases separated by comma, space or paragraph return."
-        previous_case_numbers: "You can include multiple cases separated by comma, space or paragraph return."
-        other_subject_ids: "You can include multiple cases separated by comma, space or paragraph return."
+      offender_sar:
+        prison_number: "You can include multiple numbers separated by comma or space."
+        subject_aliases: "You can include multiple aliases separated by comma or space."
+        previous_case_numbers: "You can include multiple cases separated by comma or space."
+        other_subject_ids: "You can include multiple numbers separated by comma or space."
```

Comment:
Ideally the keys should be in alphabetical order, and the quotes are not necessaryi.e.:
```
previous_case_numbers: You can include multiple cases separated by comma or space.
prison_number: You can include multiple numbers separated by comma or space.
other_subject_ids: You can include multiple numbers separated by comma or space.
subject_aliases: You can include multiple aliases separated by comma or space.  
```

You can usually re-arrange things alphabetically using your text editor e.g in sublime highlight text -> F5

Reply:
Yes, F9 in the case of Linux, done as well. Thanks for picking these up, things I overlook all the time!